### ISSUE TITLE
reactCustomScroll.js:289 Uncaught TypeError: Super expression must either be null or a function

### ISSUE THREAD

**Issue by NaveenB** [rommguy/react-custom-scroll#87]
the following error only on PROD deployment, on *apache web server* and also on *azure app service* deployment.
![image](https://user-images.githubusercontent.com/144371/84711736-4574db80-af67-11ea-8100-a7afcbbed615.png)

Works fine with local deployment with npm/ yarn start

What could be the issue?

**Comment by rommguy** [comment]
Sorry, I don't have experience with those environments. Can you share a link or something similar where I can debug this?
Maybe instructions on how to reproduce?

**Comment by NaveenB** [comment]
Thanks for the response. Currently I don't have this code not deployed anywhere. But I really like the animation part of scrollbar. 

For me to understand, how I can setup the environment for you, to reproduce the issue, a question: do you have Microsoft Office 365 account? ( As the app has integration to MS OAuth)

**Comment by rommguy** [comment]
Yes, I have office 365 account

**Comment by mygurukool** [comment]
Please click on this link  [https://custom-scroll.netlify.app](https://custom-scroll.netlify.app) and you shall see the error in browser console.

Do let me know should you need additional information. 

Thanks :)

**Comment by mygurukool** [comment]
please let me know should you need more details

**Comment by martimarkov** [comment]
Having the same issue.

**Comment by martimarkov** [comment]
@mygurukool Maybe share the code + webpack config?

**Comment by rommguy** [comment]
@mygurukool, @martimarkov 
Is this still relevant?

**Comment by EugeCos** [comment]
Yes, were getting this error on PROD after updating `react-custom-scroll` from version 4.1 to 4.2.2
Note that everything works fine on local environment.
